README.md

```diff
@@ -88,66 +67,353 @@ Finally, compile it!
 
-### Testing (Again)
-We provided you with an ERC20 test file earlier in this tutorial.
-Now it's time to test this ERC20 again.
-This time, however, we'll be testing our new OVM-compatible smart contract on top of Optimistic Ethereum.
-Luckily, this is almost as easy as compiling the contract!
+> **Side-note on the Node.js scripts:** Secretly, we hid the logic for this quick command within your `package.json`. There are actually _two_ compilation steps being run here, sequentially. First, `yarn compile` runs `yarn compile:evm` which obviously compiles your ERC20 contract and spits out an artifacts folder for the EVM. The second command being run in this step is, of course, 💃 `yarn compile:ovm` 🕺, which has a similar process, but instead uses a different naming of filepaths (which we make use of later 😉).
 
-First, make a copy of [`optimism-tutorial/test/erc20.spec.ts`](https://github.com/ethereum-optimism/optimism-tutorial/blob/main/test/erc20.spec.ts).
-You can name the copy whatever you'd like, perhaps `optimistic-erc20.spec.ts`.
-We'll modify this copy in just a minute.
+Here, `artifacts-ovm` signifies that the contracts contained in this directory have been compiled for the OVM, the **O**ptimistic **V**irtual **M**achine, as opposed to the Ethereum Virtual Machine.
 
-Now we're going to add another Hardhat plugin to [`optimism-tutorial/hardhat.config.ts`](https://github.com/ethereum-optimism/optimism-tutorial/blob/main/hardhat.config.ts):
+## Running Optimistic Ethereum locally
 
-```ts
-// hardhat.config.ts
+To simulate a development environment that is as close to mainnet Optimistic Ethereum while still being local, you're we recommend replicating the layer 1 Ethereum chain and the layer 2 Optimism chain.
+Fortunately, we have a handy dandy [integrations repo](https://github.com/ethereum-optimism/optimism-integration) all nicely dockerized for you to run your own local instance of Optimistic Ethereum!
+
+Let's get our local instance setup by running these commands:
+
+```sh
+git clone git@github.com:ethereum-optimism/optimism-integration.git --recurse-submodules
+cd optimism-integration
+./pull.sh
+```
+
+What we're doing here first is cloning the `optimism-integration` repo, which comes with a dockerized L2 chain (OVM) and a dockerzied L1 chain (EVM).
+Then, we enter the top level directory of your newly cloned (local) repo.
+Next, we run the `./pull.sh` command to pull all the docker images to start your local instance.
+
+Lastly, we'll run the `./up.sh` command to spin your docker containers up:
+
+```sh
+./up.sh
+```
+
+(NOTE: These last two commands are provided by shell scripts that we created for you 😊.)
+The containers will take some time to fully spin up, but once they do, you should see something like this flash by in the logs at some point 
+(NOTE: These logs are not important and are only shared to confirm you're running Optimistic Ethereum correctly.):
```

Comment:
General comment on writing style, we may not need to be adding in this sort of `(NOTE: ...)` style side-thoughts, and could simply write it all out in narrative paragraph-style to make it slightly more readable. For me, at least, I tend to treat things in parentheses as optional reading, so it kind of messes with my head when there's lots of them.